app/controllers/api/v1/articles_controller.rb

```diff
@@ -16,4 +23,12 @@ def show
   def render_active_record_error
     render json: { error_message: 'Sorry we can not find that article' }, status: :not_found
   end
+
+  def check_valid_category
```

Comment:
This you could also change to `valid_category?`

Reply:
Yes! We agree, Changed it now